frontend/src/Components/HomePage/User.jsx

```diff
@@ -1,22 +1,34 @@
 import React from "react";
 import { DropdownButton, Dropdown } from "react-bootstrap";
-import { openFullscreen } from "../ReusableComponents/ReusableFunctions";
+import { openFullscreen, redirectTo, onUserLogout } from "../ReusableComponents/ReusableFunctions";
+import { useNavigate } from "react-router-dom";
+import ROUTES from "../Routes/ROUTES";
 
 const User = ({ setAccessToken }) => {
-   const userLogout = () => {
-      localStorage.removeItem("accessToken");
-      setAccessToken("");
-   };
+   const navigate = useNavigate();
+   const accessToken = localStorage.accessToken;
 
    return (
       <DropdownButton align="end" title="User" id="dropdown-menu-align-end" variant="secondary">
-         <Dropdown.Item eventKey="1">Profile</Dropdown.Item>
+         <Dropdown.Item
+            onClick={() => {
+               redirectTo(navigate, ROUTES.USERINFO, 1);
+            }}
+            eventKey="1"
+         >
+            Profile
+         </Dropdown.Item>
          <Dropdown.Item eventKey="2">Settings</Dropdown.Item>
          <Dropdown.Item onClick={openFullscreen} eventKey="3">
             Fullscreen
          </Dropdown.Item>
          <Dropdown.Divider />
-         <Dropdown.Item onClick={userLogout} eventKey="4">
+         <Dropdown.Item
+            onClick={() => {
+               onUserLogout(accessToken, "accessToken", setAccessToken);
```

Comment:
why passing "accessToken" string, seems weird.

Reply:
Removed the code that requires to delete certain item from localStorage, instead i've implmented localstorage.clear() in the function since everything needs to be deleted on LogOut.
Thanks.